kit/src/elements/radio_list/mod.rs

```diff
@@ -0,0 +1,34 @@
+use crate::elements::button::Button;
+use crate::elements::Appearance;
+use common::icons::outline::Shape as Icon;
+use dioxus::prelude::*;
+
+#[derive(Props)]
+pub struct Props<'a> {
+    initial_value: String,
+    values: Vec<String>,
+    onchange: EventHandler<'a, String>,
+}
+
+#[allow(non_snake_case)]
+pub fn RadioList<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
+    let internal_state = use_state(&cx, || cx.props.initial_value.clone());
```

Comment:
Just so clippy dont complain

```suggestion
    let internal_state = use_state(cx, || cx.props.initial_value.clone());
```